aws-opentelemetry-distro/tests/amazon/opentelemetry/distro/test_instrumentation_patch.py

```diff
@@ -109,6 +108,9 @@ def _validate_patched_botocore_instrumentation(self):
         sns_attributes: Dict[str, str] = _do_extract_sns_attributes()
         self.assertTrue("aws.sns.topic_arn" in sns_attributes)
         self.assertEqual(sns_attributes["aws.sns.topic_arn"], _TOPIC_ARN)
+        sns_success_attributes: Dict[str, str] = _do_sns_on_success()
+        self.assertTrue("aws.sns.topic_arn" in sns_success_attributes)
+        self.assertEqual(sns_success_attributes["aws.sns.topic_arn"], _TOPIC_ARN)
```

Comment:
You should call `on_success` in `_validate_unpatched_botocore_instrumentation`, too.

Reply:
Done